WalletWasabi/WabiSabi/Client/DependencyGraphTaskScheduler.cs

```diff
@@ -172,6 +175,7 @@ public async Task StartOutputRegistrationsAsync(IEnumerable<TxOut> txOuts, BobCl
 						await Task.Delay(delay, cancellationToken).ConfigureAwait(false);
 					}
 					await smartRequestNode.StartOutputRegistrationAsync(bobClient, txOut.ScriptPubKey, cancellationToken).ConfigureAwait(false);
+					return txOut;
 				}
 				catch (WabiSabiProtocolException ex) when (ex.ErrorCode == WabiSabiProtocolErrorCode.AlreadyRegisteredScript)
```

Comment:
This was never executed because the exception was wrapped around an `HttpRequestException`, what was wrong.

Reply:
Good catch, thank you.